test/e2e/framework/util.go

```diff
@@ -352,6 +352,13 @@ func SkipUnlessNodeOSDistroIs(supportedNodeOsDistros ...string) {
 	}
 }
 
+// SkipIfNodeOSDistroIs skips if the node OS distro is included in the unsupportedNodeOsDistros.
+func SkipIfNodeOSDistroIs(unsupportedNodeOsDistros ...string) {
```

Comment:
i don't think the variadic string list is needed here, but looks like this is following the existing pattern.